CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+  prometheus-alertmanager                        registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:25bed531ccb0ff16ce19b927265f03cb9b2d572caa224ef302002269e925d83c
+  prometheus-config-reloader                     registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:deacbd618b3c037cc8c99a83db2c2a1053db517b0a0bfdfdeb309591559c3eea
+  prometheus-node-exporter                       registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:c199e7353642ed1a4237416055a75b0e415034c7ec48bbc8ae8d12b72552f819
+  prometheus-operator                            registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:ec28b9dc5ad9184d0d70b85e5bc618c809084b293cbc57c215bf845bf7147b2b
+```
+
+In `release new` command a list of replaced images can be specified:
+```
+$ oc adm -a /path/to/pull_secret.json \
+  release new \
+  --from-release registry.svc.ci.openshift.org/origin/release:4.4 \
+  --to-image quay.io/vrutkovs/okd-release:4.4-updated-prometheus \
+  prometheus=docker.io/prom/prometheus:v2.17.1
+```
+
+This command would do the following:
+
+* fetch `--from-release` image and parse image references in it
+* replace `prometheus` reference with a new one
+* create a new `--to-image` release with updated reference.
+
+`release new` command supports more than one replacement in the cli (separated by the space).
+
+Installer in the new image would also be updated - it would point to the new release:
+```
+$ oc adm release extract '--command-os=/usr/bin/openshift-install' --to=quay.io/vrutkovs/okd-release:4.4-updated-prometheus
+```
+This installer would now use this release as a source of truth - and use Prometheus v2.17.1 pulling it from Dockerhub.
+
+# Replacing other images
+
+Images included in the release payload are prepared on CI using Openshift builds from git repos (with one exception - `machine-os-content`, see below). In order to find out details about particular 
+image use `--commit-urls`:
+```
+$ oc adm release info registry.svc.ci.openshift.org/origin/release:4.4 --commit-urls
+...
+Images:
+  NAME                                           URL 
+  aws-machine-controllers                        https://github.com/openshift/cluster-api-provider-aws/commit/5fa82204468e71b44f65a5f24e2675dbfa0f5c29
+  azure-machine-controllers                      https://github.com/openshift/cluster-api-provider-azure/commit/832a43a30d7f00cd6774c1f5cd117aeebbe1b730
+  baremetal-installer                            https://github.com/openshift/installer/commit/e0b9dedd751543fbc01066a3049ff000e60b1459
+
+```
+This shows a particular commit used to build these images.
+
+In order to include a change in OKD release you'd need to make code changes and rebuild the image. Most repositories have `Dockerfile` at the root of the repo - so a simple `podman build` would rebuild it.
```

Comment:
~~you'd~~ you
[...]
have **a** `Dockerfile`
[...]
~~would rebuild~~ rebuilds